seacrowd/sea_datasets/vintext/vintext.py

```diff
@@ -0,0 +1,214 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Vintext is a challenging scene text dataset for Vietnamese, where some characters are equivocal in the visual form due to accent symbols.
+This dataset contains 2000 fully annotated images with 56,084 text instances. Each text instance is delineated by a quadrilateral bounding box and associated with the ground truth sequence of characters.
+The dataset is randomly split into three subsets for training (1,200 images), validation (300 images), and testing (500 images).
+"""
+import os
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+import gdown
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+_CITATION = """\
+@inproceedings{m_Nguyen-etal-CVPR21,
+    author = {Nguyen Nguyen and Thu Nguyen and Vinh Tran and Triet Tran and Thanh Ngo and Thien Nguyen and Minh Hoai},
+    title = {Dictionary-guided Scene Text Recognition},
+    year = {2021},
+    booktitle = {Proceedings of the {IEEE} Conference on Computer Vision and Pattern Recognition (CVPR)},
+}
+"""
+
+_DATASETNAME = "vintext"
+
+_DESCRIPTION = """\
+Vintext is a challenging scene text dataset for Vietnamese, where some characters are equivocal in the visual form due to accent symbols.
+This dataset contains 2000 fully annotated images with 56,084 text instances. Each text instance is delineated by a quadrilateral bounding box and associated with the ground truth sequence of characters.
+The dataset is randomly split into three subsets for training (1,200 images), validation (300 images), and testing (500 images).
```

Comment:
This split description is for the converted dataset (in COCO format), yet here you only implemented the original dataset variant. Any reason why don't you implement the converted dataset as well (maybe as a subset)? If it is difficult, I suggest replacing this description with some description saying that this dataloader only implements the original data from the homepage.

Reply:
Thank you for the suggesstion! It's really a surprise most of the reviews were my contemplations also 😂.

Hmmm, my only thinking was it's better to choose one of the format since new `subset` by definition (please correct me if i'm wrong) often has either different content / content & task / content & task & schema within the new `subset`'s dataset. (like `movie_a`, `movie_b`, and `movie_c` type of stuff, they have different content), whereas the only different between `COCO` and `original` subset is only the format (although `COCO` is obviously more detailed).

And the more crucial one is, both of the `subset` will have their very own `SEACrowd` (`seacrowd_coco`, `seacrowd_ori`, etc.) schema, which if we follow current implementation, will be exactly the same (only mapping the `x1, x2,..., xn` from rich `COCO` format). I think it's better to discuss whether to switch `image_text` schema 's format first with @holylovenia if we want to incorporate the `COCO` format.

I'll narrow down the description for now. (even if we proceed with `COCO`, I think it's better to left out the `original`? since the `original` is actually the subset / more concise version of `COCO` format)

Reply:
Well I guess i'm overcomplicating the definition of `subset`. We can implement the `COCO` format too!

Maybe let's wait for the `image_text` second opinion from @holylovenia.

Reply:
@patrickamadeus Thanks, I have replied on the comment thread above. 👍